impl/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>com.okta.commons</groupId>
             <artifactId>okta-http-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.okta.commons</groupId>
+            <artifactId>okta-http-okhttp</artifactId>
+            <scope>test</scope>
```

Comment:
Is this dependency still used at test time?

Reply:
nope. removed.